PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkToolException.kt

```diff
@@ -23,6 +23,7 @@ package com.microsoft.azure.hdinsight.common.classifiedexception
 
 import com.intellij.openapi.application.ApplicationManager
 import com.microsoft.azure.hdinsight.spark.common.YarnDiagnosticsException
+import com.microsoft.azuretools.azurecommons.helpers.AzureCmdException
```

Comment:
Seems like useless import?

Reply:
remove next iteration

---
In reply to: [244300766](https://github.com/Microsoft/azure-tools-for-java/pull/2584#discussion_r244300766) [](ancestors = 244300766)